common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -800,14 +800,18 @@ class @Problem
     attempts_remaining = @submitButton.data('attempts-remaining')
     if enable && attempts_remaining != 0
       @submitButton.removeAttr 'disabled'
+      # the is-disabled is needed for Lettuce test below
+      # lms/djangoapps/courseware/features/problems.py submit_problem(step)
       @submitButton.removeClass 'is-disabled'
       if changeText
         @submitButtonLabel.text(@submitButtonSubmitText)
     else
       @submitButton.attr({'disabled': 'disabled'})
+      # the is-disabled is needed for Lettuce test below
+      # lms/djangoapps/courseware/features/problems.py submit_problem(step)
       @submitButton.addClass 'is-disabled'
       if changeText
-        @submitButtonLabel.text(@submitButtonSubmittingText)
+        @submitButtonLabel.text(@submitButtonSubmitText)
```

Comment:
@alisan617 We don't want to lose the "Submitting" state. Can you please back this change out, and we can discuss as a team what we want to do? Note that @chris-mike made some suggestions in a comment on the PR.